src/scripts/main.js

```diff
@@ -1 +1,15 @@
 'use strict';
+
+const largeImage = document.getElementById('largeImg');
+const thumbnails = document.querySelectorAll('.gallery__img');
+
+thumbnails.forEach((thumbnail) => {
+  thumbnail.addEventListener('click', () => {
+    event.preventDefault(); // Запобігаємо стандартній дії посилання
```

Comment:
The `event` object is not defined in this scope. You should pass it as a parameter to the event listener function: `thumbnail.addEventListener('click', (event) => { ... });`